gradle.properties

```diff
@@ -1,3 +1,3 @@
 kotlin.code.style=official
-release_version=5.15.0
+release_version=6.0.0
```

Comment:
question: Do we need to increment the major version? Nothing has changed in the public API (especially considering the fact that the public API is HTTP endpoints)

Reply:
Maybe you are right